src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FallThroughCheck.java

```diff
@@ -102,17 +64,118 @@
  * To configure the check:
  * </p>
  * <pre>
- * &lt;module name=&quot;FallThrough&quot;/&gt;
+ * &lt;module name="FallThrough"/&gt;
+ * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * switch (i) {
+ *   case 1:
+ *     i++;
+ *   case 2: // violation, case 1 contains code but lacks break, return, throw or continue statement
+ *     i++;
+ *     break;
+ *   case 3: // OK
+ *     i++;
+ *     return;
+ *   case 4: // OK
+ *     i++;
+ *     throw new Exception();
+ *   case 5: // OK
+ *     i++;
+ *     continue;
+ *   case 6: // OK
+ *   case 7: // OK, case 6 does not contain code
```

Comment:
`case 6` -> previous case

Reply:
Sorry I don't understand what you meant

Reply:
I mean that do not write `case <the number>`, just write `previous case` instead of `case 6`, in next modifications it is easier to support it in case number is changed.

Reply:
Ok I see. I'll fix that

Reply:
fixed